src/components/ActionMenu/ActionMenu.spec.js

```diff
@@ -6,32 +6,32 @@ describe('Action Menu Suite', () => {
         'Action Menu',
         'Action Menu with disabled menu item %26 tooltip',
     ];
-    const singleItemStory = 'Action Menu with one menu item';
     const actionMenu = '[data-qa-action-menu]';
     const actionMenuItem = '[data-qa-action-menu-item]';
-    const actionMenuSingleItem = '[data-qa-action-menu-link]';
 
     it('should display action menu element', () => {
         navigateToStory(component, menuStories, () => {
-            browser.waitForVisible(actionMenu);
+            $(actionMenu).waitForDisplayed();
         });
     });
 
     it('should display action menu items', () => {
         executeInAllStories(component, menuStories, () => {
-            browser.click(actionMenu);
-            browser.waitForVisible(actionMenuItem);
+            $(actionMenu).click();
+            $(actionMenuItem).waitForDisplayed();
             expect($$(actionMenuItem).length).toBeGreaterThan(1);
             $$(actionMenuItem).forEach(i => expect(i.getTagName()).toBe('li'));
         });
     });
 
     it('should hide the menu items on select of an item', () => {
         navigateToStory(component, menuStories[0]);
-        browser.click(actionMenu);
-        browser.waitForVisible(actionMenuItem);
-        browser.click(actionMenuItem);
-        browser.waitForVisible(actionMenuItem, 1000, true);
+        $(actionMenu).click();
+        $(actionMenuItem).waitForDisplayed();
+        $(actionMenuItem).click();
+        
+            $(actionMenuItem).waitForDisplayed(1500, true);
```

Comment:
This one is failing for me pretty consistently.

```
[chrome 75.0.3770.100 MAC #0-0] Action Menu Suite
[chrome 75.0.3770.100 MAC #0-0]    ✓ should display action menu element
[chrome 75.0.3770.100 MAC #0-0]    ✓ should display action menu items
[chrome 75.0.3770.100 MAC #0-0]    ✖ should hide the menu items on select of an item
[chrome 75.0.3770.100 MAC #0-0]    ✓ should display tooltip menu item
[chrome 75.0.3770.100 MAC #0-0]
[chrome 75.0.3770.100 MAC #0-0] 3 passing (7.3s)
[chrome 75.0.3770.100 MAC #0-0] 1 failing
[chrome 75.0.3770.100 MAC #0-0]
[chrome 75.0.3770.100 MAC #0-0] 1) Action Menu Suite should hide the menu items on select of an item
[chrome 75.0.3770.100 MAC #0-0] Error: element ("[data-qa-action-menu-item]") still displayed after 1500ms
[chrome 75.0.3770.100 MAC #0-0] Error: element ("[data-qa-action-menu-item]") still displayed after 1500ms
[chrome 75.0.3770.100 MAC #0-0]     at UserContext.it (/Users/pthiel/Developer/manager/src/components/ActionMenu/ActionMenu.spec.js:33:31)
```

Reply:
This one is a bit odd in that even in debug mode the menu is no longer showing, however upping the wait time seems to have resolved it now.

Reply:
I was still inconsistent runs with a 3000ms wait, i bumped it to 5000ms and the tests seem to be passing consistently.. just because i was curious, i changed the assertion to use `waitForExist`:
`$(actionMenuItem).waitForExist(3000, true);` on line 32 and it passed consistently and quickly every time 😕